tests/test_model.py

```diff
@@ -373,7 +373,7 @@ def test_loss_fn():
     assert isinstance(test_cfg, DictConfig)
 
     test_cfg.device = 'cuda:0'
-    test_cfg.model.init_device = 'cuda:0'
+    test_cfg.model.init_device = 'cpu'
```

Comment:
is this to help the test be less flakey?

Reply:
yea, otherwise, with the new `init` change, it fails after fewer steps (ex. 3). I don't have a great sense of why that is,  but seems like the new version may introduce more/different non-determinism for this particular test.